omnibus/files/private-chef-cookbooks/private-chef/files/default/html/index.html

```diff
@@ -39,7 +39,7 @@ <h1>Are You Looking For the Chef Infra Server?</h1>
       <div class="mybox">
       </div>
       <div class="footer-bottom">
-        <span>&copy; 2010&thinsp;&ndash;&thinsp;2020 Chef Software, Inc. All Rights Reserved</span>
+        <span>&copy; 2010&thinsp;&ndash;&thinsp;2021 Chef Software, Inc. All Rights Reserved</span>
```

Comment:
Should this be replaced with Progress Software Corporation?

Reply:
We haven't updated to progress and since chef is part of progress it's all valid stil